### PR TITLE
3513 Fixes setuptool env inssue

### LINKED ISSUE #3513: CI test errors
I see some CI test errors in this PR, could you please help take a look?

Thanks in advance.

_Originally posted by @Nic-Ma in https://github.com/Project-MONAI/MONAI/issues/3508#issuecomment-997591075_

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes #3513

### Description
avoid v60.0.0 setuptools

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: requirements-min.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Requirements for minimal tests
 -r requirements.txt
-setuptools>=50.3.0
+setuptools>=50.3.0,!=60.0.0
```
Comment (Nic-Ma) [inline requirements-min.txt:3]: Hi @wyli ,

Thanks for your quick fix.
Could you please help describe the root cause?

Thanks in advance.

Comment (wyli) [reply requirements-min.txt:3]: I didn't look into the details but there are some discussions in the issue list https://github.com/pypa/setuptools/issues

**Comment by wyli** [comment]
looks like skipping the setuptools version 60.0 update is good enough to address #3513 @Nic-Ma

File: requirements-min.txt (hunk 1 above)
Comment (Nic-Ma) [reply requirements-min.txt:3]: Could you please help add some comments here? I am afraid we don't remember why add it in the future...
Others look good to me.

Thanks.

Comment (wyli) [reply requirements-min.txt:3]: I think we can track it with git blame and all the github ticket and pull request information. https://github.com/Project-MONAI/MONAI/blame/372f149b3472d8ef1d0a5d022c8ab34bc754b035/requirements-min.txt

**Pull request merged**